frontend/src/components/ResourcesDisplayCard.jsx

```diff
@@ -1,7 +1,11 @@
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
+import resources from "./ResourceObjects";
 import styles from "../styling/NotesDisplayCard.module.scss";
+import resourceStyles from "../styling/ResourceModal.module.scss";
 
 export default function ResourcesDisplayCard({ handleClose }) {
+  const [currentResourceId, setCurrentResourceId] = useState(resources[0].id);
```

Comment:
nit: I see that resources is a hard coded array, but this can throw an error if it is empty (which is perfectly valid say this should be fetched from a back-end)